### PR TITLE
Update index.rst

### PR DESCRIPTION
**Pull request by ykravtsov**

### CODE REVIEW

**Comment by ascott18** [comment]
Unclear on the reason for the change to "birthBirth". Export change is good, though.

**Comment by ascott18** [comment]
Oh, I see - in the C# model it was called birthDate. Could you make the correction to your branch?

File: docs/index.rst [hunk 1]
```diff
@@ -114,7 +114,7 @@ We can create a details page for a Person by creating:
                 <dd data-bind="text: name"></dd>
 
                 <dt>Date of Birth </dt>
-                <dd data-bind="moment: dateOfBirth, format: 'MM/DD/YYYY hh:mm a'"></dd>
+                <dd data-bind="moment: birthBirth, format: 'MM/DD/YYYY hh:mm a'"></dd>
```
Comment (ascott18) [inline docs/index.rst]: Change to birthDate

**Comment by ykravtsov** [comment]
:) glad you saw what I meant don't know how I messed that up

**Pull request merged**